src/js/routes/Values/ValuesList.jsx

```diff
@@ -28,8 +30,22 @@ export default class ValuesList extends Component {
   componentDidMount () {
     IssueActions.retrieveIssuesToFollow();
     this.issueStoreListener = IssueStore.addListener(this.onIssueStoreChange.bind(this));
+
+    this.setState({ currentIssue: this.props.currentIssue });
+  }
+
+  componentWillReceiveProps (nextProps) {
+    this.setState({ currentIssue: nextProps.currentIssue });
   }
 
+  shouldComponentUpdate (nextState) {
+    if (this.state.currentIssue !== nextState.currentIssue) {
```

Comment:
I would suggest comparing the currentIssue weVoteId here, instead of the whole object.